test/multi.js

```diff
@@ -8,18 +8,18 @@ describe("multi", () => {
       apicalypse()
         .query("games", "latest-games")
         .fields("name")
-        .where(`created_at < ${now}`)
-        .sort("created_at desc"),
+        .sort("created_at desc")
+        .where(`created_at < ${now}`),
       apicalypse()
         .query("games", "coming-soon")
         .fields("name")
-        .where(`created_at > ${now}`)
         .sort("created_at asc")
+        .where(`created_at > ${now}`),
     ]);
 
-    assert(
-      test.apicalypse ===
-        `query games "latest-games" { fields name;where created_at < ${now};sort created_at desc; };query games "coming-soon" { fields name;where created_at > ${now};sort created_at asc; };`
+    assert.deepStrictEqual(
+      test.apicalypse,
+      `query games "latest-games" { fields name;sort created_at desc;where created_at < ${now}; };query games "coming-soon" { fields name;sort created_at asc;where created_at > ${now}; };`
```

Comment:
Wait... what? We support multiple queries in a single request?
How does this data come back? Is this ever used?

Reply:
The data comes back in named fields. You can read about it here: https://api-docs.igdb.com/#multi-query